internal/provider/resource_vault_cluster.go

```diff
@@ -169,17 +174,15 @@ func resourceVaultClusterCreate(ctx context.Context, d *schema.ResourceData, met
 
 	publicEndpoint := d.Get("public_endpoint").(bool)
 
-	// TODO: Tier is hard-coded for now, but eventually will be required input on the resource.
-	tier := vaultmodels.HashicorpCloudVault20201125TierDEV
-
 	log.Printf("[INFO] Creating Vault cluster (%s)", clusterID)
 
 	vaultCuster := &vaultmodels.HashicorpCloudVault20201125InputCluster{
 		Config: &vaultmodels.HashicorpCloudVault20201125InputClusterConfig{
 			VaultConfig: &vaultmodels.HashicorpCloudVault20201125VaultConfig{
 				InitialVersion: vaultVersion,
 			},
-			Tier: tier,
+			//TODO: HashicorpCloudVault20201125Tier still hardcoded as in consul
```

Comment:
What do you mean here?

Reply:
I just don't quite understand why `$(date +%Y%m%d)` is in the name of the tier type. Wouldn't a fresh list cause more code update in the future instead of a generalised one? Maybe I'm just missing something :-)

Reply:
Oh I see. Yeah we're still tuning our SDK generation, so hoping to resolve that issue!